django/test/client.py

```diff
@@ -441,14 +441,14 @@ def __init__(self, enforce_csrf_checks=False, **defaults):
 
     def store_exc_info(self, **kwargs):
         """
-        Stores exceptions when they are generated by a view.
+        Store exceptions when they are generated by a view.
         """
         self.exc_info = sys.exc_info()
 
     @property
     def session(self):
         """
```

Comment:
Should these by one liners ? e.g. """Obtain the current session variables.""" -- we currently don't have consistency about that.